src/diagnostics/wizardSteps/EligibilityCheck.ts

```diff
@@ -27,11 +27,16 @@ export class EligibilityCheck extends WizardStep {
             throw new Error('Cannot read "linuxFxVersion"');
         }
 
-        const [framework, imageName] = linuxFxVersion.split('|');
-        const enabledImages = vscode.workspace.getConfiguration('appService').get<string[]>('enabledDockerImages');
-
-        if ('docker' !== framework.toLocaleLowerCase() || enabledImages.indexOf(imageName.toLocaleLowerCase()) === -1) {
-            throw new Error(`Please use one of the supported docker image. The ${framework}|${imageName} combination is not supported`);
+        const [framework, fullImageName] = linuxFxVersion.split('|');
+        // Remove the 'tag' portion of the image name.
+        const imageName = fullImageName.split(':')[0];
```

Comment:
Is it possible to the array to be empty here?

Reply:
I don't think so. Above we tested site.kind to filter out windows, so when here it is linux, the linuxFxVersion will have an value we need.